packages/hardhat/.prettierrc

```diff
@@ -1,5 +1,4 @@
 {
-  "plugins": ["prettier-plugin-solidity"],
```

Comment:
why we delete this? does this kill the solidity specific formatting?

Reply:
this was causing my local to break for some reason. I did not want to commit it. I will revert this

Reply:
this line, causes prettier to break for @KMKoushik as well. Can we remove this plugin? what is its use?